diff --git a/CMakeLists.txt b/CMakeLists.txt
index 693bab6..770ba85 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -145,7 +145,7 @@ the command \"git submodule update --init --recursive\" in the dune-logging sour
     dune-logging-fmt
     PUBLIC
       $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/dune/vendor/fmt>
-      $<INSTALL_INTERFACE:dune/vendor/fmt>
+      $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/dune/vendor/fmt>
     )
 
 endif()
